web3.js/test/connection.test.ts

```diff
@@ -186,71 +186,59 @@ describe('Connection', () => {
   });
 
   it('get multiple accounts info', async () => {
-    const account1 = Keypair.generate();
-    const account2 = Keypair.generate();
 
-    {
-      await helpers.airdrop({
-        connection,
-        address: account1.publicKey,
-        amount: LAMPORTS_PER_SOL,
-      });
+    const multipleAccounts = [];
+
+    for (let i = 0; i < 100; i++) {
+      multipleAccounts.push(Keypair.generate());
+    }
```

Comment:
Since you're creating _exactly_ 100 accounts, this test isn't testing all of your chunking behavior!

What would happen if you created just one more than the limit? You'd probably need to mock _two_ RPC calls, to verify that both calls were actually made.